tests/ws_integration/expected/nim.cfg

```diff
@@ -5,14 +5,14 @@
 --path:"karax"
 --path:"sass/src"
 --path:"nimcrypto"
---path:"nim-markdown/src"
+--path:"markdown/src"
```

Comment:
but the package's name is nim-markdown!

Reply:
That's a tricky one. The package repo's url and git's default dir is `nim-markdown/` but the package manager name is `markdown` both in it's Nimble file, packages.json, and in Nim itself.

I changed it primarily to handle cases with duplicate names like `redis` and `zedeus/redis`. Using the package's "proper" name allows better checking for duplicates with the packages.json and in case someone renames the Git URL after forking it. 

But yes, it would be a bit of a change of how Atlas clones to the target dir.